meilisearch-http/src/routes/indexes/documents.rs

```diff
@@ -274,3 +274,20 @@ pub async fn clear_all_documents(
     debug!("returns: {:?}", task);
     Ok(HttpResponse::Accepted().json(task))
 }
+
+/// Returns the Content-Type header value without the charset=utf-8 or anything,
+/// only the value that is before the first semicolon (;).
```

Comment:
Is it forbidden to put the charset as the first value?
```
charset=utf-8;type=application/json
```

Reply:
Do you mean `charset=utf-8; application/json`? It is very uncommon to see this so I decide to throw them out in this case.

Reply:
I would prefer that we use [this method](https://docs.rs/actix-web/3.3.2/actix_web/trait.HttpMessage.html#method.mime_type) that is implemented on `HttpRequest`, and leverage the `Mine` library, instead of doing this ourselves

Reply:
It seems, [according to RFC 7231](https://datatracker.ietf.org/doc/html/rfc7231#section-3.1.1.1), that the media type is always before anything else (`charset`).